### PR TITLE
Add Github workflows for automatic builds and testing on PRs

### PR DESCRIPTION
**Pull request by acnebs**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "test": "jest",
     "test-e2e": "yarn build && node -r @babel/register ./config/scripts/CheckBuildsExist.js && cp -R ./build/* ./src/electron && cross-env NODE_ENV=test testcafe electron:./src/electron ./tests-e2e && rm -rf ./src/electron/dist ./src/electron/main.js",
     "tsc": "cross-env NODE_ENV=development ./node_modules/.bin/tsc",
-    "checks": "concurrently 'yarn lint' 'yarn test' 'yarn tsc'",
+    "checks": "concurrently \"yarn lint\" \"yarn test\" \"yarn tsc\"",
```
Comment (lenilsonjr) [inline package.json:11]: should we include the e2e tests here? Or are they too much trouble to setup in CI for now?

Comment (acnebs) [reply package.json:11]: Yep that's a good call. Will add.

File: .github/workflows/pr-to-dev.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        uses: DevExpress/testcafe-action@latest
+        env:
+          NODE_ENV: test
+        with:
+          args: 'electron:./src/electron ./tests-e2e'
+
+      - name: Remove files for E2E tests (!macOS)
+        if: matrix.os != 'macos-latest'
+        shell: bash
+        run: rm -rf ./src/electron/dist ./src/electron/main.js
```
Comment (lenilsonjr) [inline .github/workflows/pr-to-dev.yml:50]: 🎉

File: package.json [hunk 3]
```diff
@@ -6,9 +6,10 @@
     "lint": "cross-env NODE_ENV=development && eslint . --cache --ext .js,.jsx,.ts,.tsx",
     "lint:fix": "cross-env NODE_ENV=development eslint . --cache --ext .js,.jsx,.ts,.tsx --fix",
     "test": "jest",
-    "test-e2e": "yarn build && node -r @babel/register ./config/scripts/CheckBuildsExist.js && cp -R ./build/* ./src/electron && cross-env NODE_ENV=test testcafe electron:./src/electron ./tests-e2e && rm -rf ./src/electron/dist ./src/electron/main.js",
+    "test-e2e:no-build": "node -r @babel/register ./config/scripts/CheckBuildsExist.js && cp -R ./build/* ./src/electron && cross-env NODE_ENV=test testcafe electron:./src/electron ./tests-e2e && rm -rf ./src/electron/dist ./src/electron/main.js",
```
Comment (lenilsonjr) [inline package.json:9]: do we need this new command since we're running testcafe directly in the actions? might be useful to leave it here though.

**Review by lenilsonjr — APPROVED** [review]
looks good to me!

**Pull request merged**